detections/endpoint/executable_file_written_in_administrative_smb_share.yml

```diff
@@ -12,11 +12,22 @@ description: The following analytic identifies executable files (.exe or .dll) b
   endpoints. Red Teams and adversaries alike may abuse administrative shares for lateral
   movement and remote code execution. The Trickbot malware family also implements
   this behavior to try to infect other machines in the infected network.
-search: '`wineventlog_security` EventCode=5145 Relative_Target_Name IN ("*.exe","*.dll")
-  Object_Type=File Share_Name IN ("\\\\*\\C$","\\\\*\\IPC$","\\\\*\\admin$") Access_Mask=
-  "0x2" | stats min(_time) as firstTime max(_time) as lastTime count by EventCode
-  Share_Name Relative_Target_Name Object_Type Access_Mask user src_port Source_Address
-  | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)` | `executable_file_written_in_administrative_smb_share_filter`'
+search: '`wineventlog_security` 
+    EventCode=5145 
```

Comment:
looks like there is an extra newline char here.

Reply:
Ah, added that extra line for ease of readability. Force of habit :)

Reply:
Went ahead and removed the newline char, so it looks like this when copy/pasted:

<img width="932" alt="image" src="https://user-images.githubusercontent.com/91983279/163591753-963b4971-9d45-4ff5-810b-6e2c530e9583.png">